libraries/SRV_Channel/SRV_Channel.cpp

```diff
@@ -216,8 +216,27 @@ uint16_t SRV_Channel::get_limit_pwm(Limit limit) const
 // return true if function is for a multicopter motor
 bool SRV_Channel::is_motor(SRV_Channel::Aux_servo_function_t function)
 {
-    return ((function >= SRV_Channel::k_motor1 && function <= SRV_Channel::k_motor8) ||
-            (function >= SRV_Channel::k_motor9 && function <= SRV_Channel::k_motor12));
+    switch (function) {
+    case Aux_servo_function_t::k_motor1:
+    case Aux_servo_function_t::k_motor2:
```

Comment:
You don't need to make the change - but have a look at the use of `...` in `get_motor_num`.  It's a nice pattern to reuse in this method.

Reply:
I considered that and tried it to discover it magically consumers the same flash space. Computer optimizations FTW.

Reply:
I fixed it up with the ... along with another one right below it